projects/make-a-vis/src/lib/visualization-view/main/main.component.scss

```diff
@@ -18,6 +18,7 @@
     width: 100%;
     border: 1px solid transparent;
     background-color: white;
+    overflow: auto;
```

Comment:
Did you check if this broke the other visualizations? Seems like it might break stuff.